pkg/container-disk/container-disk.go

```diff
@@ -179,6 +179,14 @@ func GenerateContainers(vmi *v1.VirtualMachineInstance, podVolumeName string, bi
 func generateContainersHelper(vmi *v1.VirtualMachineInstance, podVolumeName string, binVolumeName string, isInit bool) []kubev1.Container {
 	var containers []kubev1.Container
 
+	const (
+		Size1M   = "1M"
+		Size10M  = "10M"
+		Size50M  = "50M"
+		Size40M  = "40M"
+		Size100M = "100M"
```

Comment:
This actually makes it look like it's all the same units, while 10m and 100m you've replaced for CPU values, are in fact `milliCPU` units and memory is measured mebibytes, I guess it all just invites confusion.

Reply:
I would either avoid doing this or use explicit units.